protos/schedules.proto

```diff
@@ -2,12 +2,36 @@ syntax="proto3";
 
 package dataform.schedules;
 
+
+message EmailNotificationChannel {
+  repeated string emails = 1;
+}
+
+message SlackNotificationChannel {
+  string webhook_url = 1;
+}
+
+message NotificationChannel {
+ string name = 1;
+ oneof channel {
+    EmailNotificationChannel email_notification = 1;
+    SlackNotificationChannel slack_notification = 2;
+  } 	
+}
+
+message ScheduleNotification {
```

Comment:
@lewish now's the right time to make the final decision about whether we have a `schedules` field in here or not.

Reply:
Personally, I'm happy with this...

Reply:
ok!